brokenLinkReport.html

```diff
@@ -0,0 +1,20 @@
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/design-system/src/components/web-components/README.md" target="_blank">design-system/packages/design-system/src/components/web-components/README.md</a></h4><div style="margin-left:.5rem"><a href="https://design.cms.gov/cdn/design-system/X.X.X/css/index.css" target="_blank">https://design.cms.gov/cdn/design-system/X.X.X/css/index.css</a> <p>Code: 404</p></div><br/></div>
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/design-system/src/components/web-components/README.md" target="_blank">design-system/packages/design-system/src/components/web-components/README.md</a></h4><div style="margin-left:.5rem"><a href="https://design.cms.gov/cdn/design-system/X.X.X/css/core-theme.css" target="_blank">https://design.cms.gov/cdn/design-system/X.X.X/css/core-theme.css</a> <p>Code: 404</p></div><br/></div>
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/ds-medicare-gov/README.md" target="_blank">design-system/packages/ds-medicare-gov/README.md</a></h4><div style="margin-left:.5rem"><a href="https://design.cms.gov/getting-started/developers/installation/?theme=medicare" target="_blank">https://design.cms.gov/getting-started/developers/installation/?theme=medicare</a> <p>Code: 404</p></div><br/></div>
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/docs/content/components/tabs.mdx" target="_blank">design-system/packages/docs/content/components/tabs.mdx</a></h4><div style="margin-left:.5rem"><a href="http://heydonworks.com/practical_aria_examples/#tab-interface" target="_blank">http://heydonworks.com/practical_aria_examples/#tab-interface</a> <p>Code: 404</p></div><br/></div>
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/docs/content/guidelines/accessibility.mdx" target="_blank">design-system/packages/docs/content/guidelines/accessibility.mdx</a></h4><div style="margin-left:.5rem"><a href="http://www.access-board.gov/508.htm" target="_blank">http://www.access-board.gov/508.htm</a> <p>Code: 404</p></div><br/></div>
+<div><h4>Filepath: <a href="/Users/jackryan/Desktop/design-system/packages/docs/content/getting-started/for-developers.mdx" target="_blank">design-system/packages/docs/content/getting-started/for-developers.mdx</a></h4><div style="margin-left:.5rem"><a href="https://design.cms.gov/cdn/<package-name>/<version>/<resource>" target="_blank">https://design.cms.gov/cdn/<package-name>/<version>/<resource></a> <p>Code: 404</p></div><br/></div>
```

Comment:
It works in the browser, but when I try to format this file with Prettier, the following URL breaks HTML parsing:

```
https://design.cms.gov/cdn/<package-name>/<version>/<resource>
```

This URL that it found is a special case, because it actually isn't a link at all but [a URL inside an example code block](https://github.com/CMSgov/design-system/blob/main/packages/docs/content/getting-started/for-developers.mdx?plain=1#L71).

Reply:
I added this file to the `.prettierignore` file in a later commit. I don't think this file should be linted.

Reply:
Just bringing it to your attention that that URL that isn't a link is getting incorrectly picked up and that it can break HTML parsing 👍

Reply:
Yeah, maybe will fix in the future. Thanks for pointing out!